docs/devices_api.yml

```diff
@@ -1,99 +1,121 @@
 
-        This method simultaneously acts as a bootstraping one - upon the first
-        auth request, the device is enrolled in the system, and we start
-        tracking its state via generated Device ID.
-
-        The returned JWT is
-          * signed with the identity service's private key
-          * encrypted with the device's public key
-
-        JWTs are issued with the following standard fields:
-          * 'exp' - expiry date
-          * 'sub' - subject (Mender-generated device ID)
-          * 'jit' - token's unique identifier
-
-        Each request is signed with the device's private key. The 'X-MEN-Signature' custom header carries the signature computed as:
-
-           BASE64(SIGN(privkey, SHA256(body)))
-
-        An increasing 'seq_no' is used to prevent replay attacks. Upon the very first request
-        the client can provide any non-zero uint64, which will be recorded by the server for subsequent requests.
+        Note that the very first authentication request from a given device ('bootstrap' request) always
+        results in 'HTTP 401 Unauthorized'. At the same time, the identity data is recorded for
+        later inspection by the user, who then explicitly accepts or rejects the device via the web GUI (backed by
+        the Device Admission Service). A subsequent authentication request will reflect this decision.
 
+        Note that in case of JWT expiration, the authentication request must be sent again.
       parameters:
         - name: auth_request
           in: body
-          description: Auth request
+          description: Authentication request.
           required: true
           schema:
-            $ref: "#/definitions/AuthRequestNew"
+            $ref: "#/definitions/AuthRequest"
         - name: X-MEN-Signature
           in: header
+          description: |
+            Request signature, computed as 'BASE64(SIGN(device_private_key, SHA256(request_body)))'.
+            Verified with the public key presented by the device.
+          required: true
           type: string
-          description: Base64 version of body signed with device's private key.
       responses:
         200:
-          description: Authentication successful - a new JWT is issued and returned
-                       (encrypted with device's public key).
-          schema:
-            type: number
-            format: binary
+          description: |
+            Authentication successful - a new JWT is issued and returned.
+            The JWT is signed with the API's private key ('RS256' signing algorithm), and contains
+            the following standard claims:
+            * 'iss' - issuer
+            * 'exp' - expiry date
+            * 'sub' - subject (auto-generated device ID)
+            * 'jti' - token's unique identifier (tracked for the purpose of revocation)
+          examples:
+              application/json:  eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.
```

Comment:
Since it is returning a blob of text, could this be text/plain, rather than application/json?